credentials/apps/core/models.py

```diff
@@ -19,6 +19,17 @@
 
 class SiteConfiguration(models.Model):
     site = models.OneToOneField(Site, null=False, blank=False)
+    edx_org_short_name = models.CharField(
```

Comment:
I believe we well need to add the migration as well.

Reply:
Done